src/components/WorkflowParams.tsx

```diff
@@ -0,0 +1,89 @@
+"use client";
+
+import { useEffect, useState } from "react";
+import { UseFormReturn } from "react-hook-form";
+import { useRouter } from "next/router";
+import { useWorkflows } from "@/context/DBContext";
+import FormProvider from "@/components/form/FormProvider";
+import { InputParams, WorkflowParams } from "@/models/workflow";
+import ParamAccordionGroup from "@/components/ParamsAccordion";
+
+export default function WorkflowParamsPage({
+  onSubmit,
+  methods
+}: {
+  onSubmit: (formValues: any) => void;
+  methods: UseFormReturn<any>;
+}) {
+  const context = useWorkflows();
+  const workflows = context?.workflows;
```

Comment:
You should be able to use object unpacking syntax here I think e.g. `const {workflows} = useWorkflows()`. Might need some extra processing if the context can be `null`, but will this component even work at all if the context is `null`? Might be worth rewriting this in a way that assumes the context won't be null.